config/logging.php

```diff
@@ -112,6 +112,7 @@
             'handler' => \Rollbar\Laravel\MonologHandler::class,
             'access_token' => env('ROLLBAR_TOKEN'),
             'level' => env('ROLLBAR_LEVEL', 'error'),
+            'code_version' => config('version.full_app_version'),
```

Comment:
Are we sure we have access to that variable from within a config file? If I recall, those files get loaded before the rest of the framework so we always have to use `env()`. (I could 100% be wrong tho. I'll test today)